soluciones/tema3/simpleAuthModule/app.js

```diff
@@ -0,0 +1,78 @@
+/* Dependencies */
+
+var express = require("express")
+  , env = process.env.NODE_ENV || "development"
+  , auth = require("./simpleauth")
+  , logger = require("morgan")
+  , bodyParser = require("body-parser")
+  , cookieParser = require("cookie-parser")
+  , cookieSession = require("cookie-session")
+
+var users = [
+  {
+    id: 0,
+    username: "uno@test.com",
+    pass: "asdf"
```

Comment:
what about creating something more readable and enterprisey? like... node@redradix.com : node :dancer: